examples/use_plugins.py

```
@@ -0,0 +1,127 @@
+import json
+from pathlib import Path
+
+# Add the parent directory to the Python path
+sys.path.insert(0, str(Path(__file__).parent.parent))
+
+from pyprocessor.utils.core.plugin_manager import (
+    get_plugin_manager,
+    discover_plugins,
+    load_plugin,
+    enable_plugin,
+    get_plugin,
```

Ruff (0.8.2)

`pyprocessor.utils.core.plugin_manager.get_plugin` imported but unused

Remove unused import: `pyprocessor.utils.core.plugin_manager.get_plugin`

(F401)